src/keymap.cpp

```diff
@@ -58,6 +58,7 @@ static OpDesc opdescs[] = {
 		_("Mark all above as read"),
 		KM_ARTICLELIST},
 	{OP_SAVE, "save", "s", _("Save article"), KM_ARTICLELIST | KM_ARTICLE},
+	{OP_SAVEALL, "save-all", "S", _("Save articles"), KM_ARTICLELIST},
```

Comment:
I'm still hoping to implement [a better keybind system](https://github.com/akrennmair/newsbeuter/issues/493) that will let us tag items and perform operations over all of them (like `tag-thread`/`tag-entry`/`tag-pattern` and `tag-prefix` in Mutt). That new system will obviate the need for custom "X for all" commands like "mark all read" and this one. As a result, I don't want to add default bindings for such commands. Please replace `"S"` with `""`, and remove this operation from `hints` array in src/itemlistformaction.cpp.

Reply:
Okay! One thing though: removing the operation and all that, how will I be able to execute it? Through configuration?

Reply:
Yeah, just add a binding:
```
bind-key S save-all articlelist
```

Removing the entry from `hints` will only affect the line at the bottom of the dialog, one that says stuff like "q:Quit ENTER:Open s:Save". The operation will still be present in the help dialog, and I'll also mention it in the changelog so people know about it.

Reply:
Ok :+1: